### ISSUE TITLE
Issues when building golos dapp contracts

### ISSUE THREAD

**Issue by zxcat** [GolosChain/cyberway#52]
Replaced eos installation with cyberway installation.
Build fails, something wrong with headers when building golos dapp contracts and tests:
```
[build] In file included from eos/golos-smart/tests/golos.vesting_tests.cpp:1:
[build] In file included from eos/golos-smart/tests/golos_tester.hpp:3:
[build] In file included from /usr/local/eosio/include/eosio/testing/tester.hpp:2:
[build] In file included from /usr/local/eosio/include/eosio/chain/controller.hpp:8:
[build] In file included from /usr/local/eosio/include/eosio/chain/account_object.hpp:11:
[build] /usr/local/eosio/include/eosio/chain/multi_index_includes.hpp:14:10: fatal error: 'multi_index.hpp' file not found
[build] #include "multi_index.hpp"
[build]          ^~~~~~~~~~~~~~~~~
```

Looks like this: https://github.com/GolosChain/cyberway/commit/75eeea5cce872a10c5a96abf33a90f0d60f00797#diff-fa226291a0e4822bd9538b92abbf177fR14 and file links caused problems

**Comment by zxcat** [comment]
More info: installed cyberway (with `eosio_install.sh`, CDT requires it) places includes in different way comparing with repo. so files linked this way: https://github.com/GolosChain/cyberway/commit/75eeea5cce872a10c5a96abf33a90f0d60f00797#diff-38bb81b5545ddadb9b25d2fec840c3e0R1 point to nowhere

**Comment by afalaleev** [comment]
Closed in #45
